### DISCUSSION TITLE
Grease Pencil Pass for Malt

### DISCUSSION THREAD

**Discussion by Psi25** [bnpr/Malt#433] [Feedback & Feature Requests]
Is there a function to have Grease Pencil as a pass input for Malt? It would be nice to have it as a pass. Having it as a pass could mean it can be used for Screen Shaders. 

Overall workflow versatility could be improved since Malt has no way of marking edges as freestyle which the Grease Pencil modifier can do.

**Comment by pragma37** [comment]
This is not really doable at the moment.
Grease Pencil is rendered inside its own engine as an overlay, there's no way to get access to it from an addon.

OTH, Freestyle Edges and geometry-based Crease and Contour line detection are something I've been working on, so that will be released sooner or later.

**Reply by Psi25** [reply]
That sounds good. I'll just have to wait.
